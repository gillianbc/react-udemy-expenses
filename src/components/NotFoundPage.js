// Stateless functional component
import { Link } from "react-router-dom";
import React from "react";

// The link here is client-side routing.  If we had used <a href="/">Go home</a> it would have been server-side routing
const NotFoundPage = () => (
  <div>
    Page not found 404 - <Link to="/">Go to the home page</Link>
  </div> )

export default NotFoundPage