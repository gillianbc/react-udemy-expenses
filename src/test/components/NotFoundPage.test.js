import React from 'react'
import { shallow } from 'enzyme'
import NotFoundPage from "../../components/NotFoundPage";
/*
* Tests the snapshot of the NotFoundPage component
* */

it('The snapshot should match when there is expense data', () => {
    // Remember, these curlies are not object brackets, they're JSX brackets
    const wrapper = shallow(<NotFoundPage />);
    expect(wrapper).toMatchSnapshot();
})



