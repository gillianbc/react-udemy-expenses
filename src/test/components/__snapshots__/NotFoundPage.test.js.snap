// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The snapshot should match when there is expense data 1`] = `
<div>
  Page not found 404 - 
  <Link
    replace={false}
    to="/"
  >
    Go to the home page
  </Link>
</div>
`;
